src/transformers/models/seaformer/modeling_seaformer.py

```diff
@@ -0,0 +1,764 @@
+        super(SeaformerInvertedResidual, self).__init__()
+        self.stride = stride
+        self.expand_ratio = expand_ratio
+        assert stride in [1, 2]
+
+        if activations is None:
+            activations = nn.ReLU
+
+        hidden_dim = int(round(inp * expand_ratio))
+        self.use_res_connect = self.stride == 1 and inp == oup
+
+        layers = []
+        if expand_ratio != 1:
+            # pw
+            layers.append(SeaformerConv2d_BN(inp, hidden_dim, ks=1))
+            layers.append(activations())
+        layers.extend([
+            # dw
+            SeaformerConv2d_BN(hidden_dim, hidden_dim, ks=ks, stride=stride, pad=ks // 2, groups=hidden_dim),
+            activations(),
+            # pw-linear
+            SeaformerConv2d_BN(hidden_dim, oup, ks=1)
+        ])
+        self.conv = nn.Sequential(*layers)
+        self.out_channels = oup
+        self._is_cn = stride > 1
+
+    def forward(self, x):
+        if self.use_res_connect:
+            return x + self.conv(x)
+        else:
+            return self.conv(x)
+
+
+class SeaformerStackedMV2Block(nn.Module):
+    def __init__(
+            self,
+            cfgs,
+            stem,
+            inp_channel=16,
+            activation=nn.ReLU,
+            width_mult=1.):
+        
+        super().__init__()
+        
+        self.stem = stem
+        if stem:
+            self.stem_block = nn.Sequential(
+                SeaformerConv2d_BN(3, inp_channel, 3, 2, 1),
+                activation()
+            )
+            
+        self.cfgs = cfgs
+        self.layers = []
+        
+        for i, (k, t, c, s) in enumerate(cfgs):
+            output_channel = _make_divisible(c * width_mult, 8)
+            exp_size = t * inp_channel
+            exp_size = _make_divisible(exp_size * width_mult, 8)
+            layer_name = 'layer{}'.format(i + 1)
+            layer = SeaformerInvertedResidual(inp_channel, output_channel, ks=k, stride=s, expand_ratio=t, activations=activation)
+            self.add_module(layer_name, layer)
+            inp_channel = output_channel
+            self.layers.append(layer_name)
+
+    def forward(self, x):
+        if self.stem:
+            x = self.stem_block(x)
+        for i, layer_name in enumerate(self.layers):
+            layer = getattr(self, layer_name)
+            x = layer(x)
+        return x
+
+    
+class SeaformerSqueezeAxialPositionalEmbedding(nn.Module):
+    def __init__(self, dim, shape):
+        super().__init__()
+        
+        self.pos_embed = nn.Parameter(torch.randn([1, dim, shape]))
+
+    def forward(self, x):
+        B, C, N = x.shape
+        x = x + F.interpolate(self.pos_embed, size=(N), mode='linear', align_corners=False)
+        
+        return x
+    
+class h_sigmoid(nn.Module):
+    def __init__(self, inplace=True):
+        super(h_sigmoid, self).__init__()
+        self.relu = nn.ReLU6(inplace=inplace)
+
+    def forward(self, x):
+        return self.relu(x + 3) / 6
```

Comment:
Is this activation function specific to SeaFormer? In either case, you can add this class to the transformers activations file and import it:
https://github.com/huggingface/transformers/blob/main/src/transformers/activations.py

See an example over here:
https://github.com/huggingface/transformers/blob/main/src/transformers/models/maskformer/modeling_maskformer.py#L551